CITA location updates (Jan. 16)

jiannis opened the pull request:
See [Pension Wise - CitA New Locations, Amendments & Removals (08012015) v2](https://docs.google.com/spreadsheets/d/1wagcFk6KvxVF93PkMJt7VFtgpe7S359IIHwK7S9DOAg/edit?usp=sharing)
- 14 locations removed
- 11 locations amended
- 30 locations added

Commit:
CITA location amendments (Jan. 16)

Commit:
CITA location removals (Jan. 16)

Commit:
CITA new locations (Jan. 16)

jiannis commented:
@andrewgarner do you have any idea why for example [Buxton](https://github.com/guidance-guarantee-programme/locator/blob/master/public/js/cita.json#L1062) has a phone number in the locator but no corresponding entry in the switchboard?

jiannis commented:
@andrewgarner ah, this guidance-guarantee-programme/switchboard@e53a5c6c explains why some numbers in the locator have no corresponding entry in the switchboard

andrewgarner commented on public/js/cita.json:
```diff
@@ -2930,11 +2870,11 @@
   },
   {
     "title": "Hertford",
```
Should we now call this location "Ware"?

andrewgarner commented on public/js/cita.json:
```diff
@@ -4101,21 +4021,21 @@
   {
     "title": "Kirkby-in-Ashfield",
-    "address": "Portland Street\nKirkby-in-Ashfield\nNottinghamshire\nNG17 7AE",
+    "address": "Ashfield Health and Wellbeing Centre\nPortland Street\nKirby in Ashfield\nNottinghamshire\nNG17 7AE",
```
The title says "Kirkby-in-Ashfield" but in the address we say "Kirby in Ashfield" – no dashes and a missing 'k'. Based on their website I think the value we had before was correct - http://ashfieldca.org.uk

Are we sure the new lat and long are required given the address is essentially the same, the only new detail is the name of the building.

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+  },
+  {
+    "title": "Ribble Valley Homes",
+    "address": "De Lacy House\nStation Road\nClitheroe\nLancashire BB7 2JT",
```
Missing a new line character before the postcode.

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "address": "Stublee Hall\nStublee Park\nBacup\nLancashire\nOL13 ODE",
+    "phone": null,
+    "hours": null,
+    "lat": null,
```
We need a lat and log for this.

jiannis replied:
Good spot! [Close to the edge](https://www.google.co.uk/maps/place/Close+To+The+Edge/@53.6927199,-2.2040092,266m/data=!3m1!1e3!4m2!3m1!1s0x0000000000000000:0xf00c8a3791f43fb8). Literally

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "b63e0799-2161-482f-bdfe-5188f904da32"
+  },
+  {
+    "title": "Citizens Advice Leicestershire (Wigston)",
```
We shouldn't include the name of the organisation in the title of the location. In research we saw this was decisive with our users. When the location is displayed on the Pension Wise website we explain the relationship with Citizens Advice.

As it stands it would read:

```
Your appointment will be at Citizens Advice Leicestershire (Wigston) Citizens Advice with a trained Pension Wise guidance specialist.
```

I would suggest a title of "Wigston" so that it reads:

```
Your appointment will be at Wigston Citizens Advice with a trained Pension Wise guidance specialist.
```

jiannis replied:
:ok_hand:

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "18ef0b7d-b13a-4284-9765-aa888aa4ba09"
+  },
+  {
+    "title": "Citizens Advice Nottingham and District",
```
We should be giving a title of the location not a title of the organisation. As it stands this would be confusing when displayed on the website:

```
Your appointment will be at Citizens Advice Nottingham and District Citizens Advice with a trained Pension Wise guidance specialist.
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
     "booking_location_id": "b74377da-d06b-48d0-9d74-cc362bbb53b3"
+  },
+  {
+    "title": "Poyton Civic Hall",
```
I would suggest a title of "Poyton" and add the fact it is at the civic hall to the address.

jiannis replied:
I wasn't sure at all about this place. Google maps [displays it](https://www.google.co.uk/maps/place/Poynton+Civic+Hall/@53.3494361,-2.1196398,17z/data=!3m1!4b1!4m2!3m1!1s0x487a4b160cb25be9:0x7f7b5dfa973c3db8) as Poy**n**ton..

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "df38d9b9-e9a9-4bf0-b56b-4309965b815c"
+  },
+  {
+    "title": "Peterlee Macmillan Information Centre",
```
Again, I would set the title to "Peterlee" and set "Peterlee Macmillan Information Centre" as the first line of the address.

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "Bishop Auckland Hospital",
```
"Bishop Auckland" and ""Bishop Auckland Hospital" as the first line of the address.

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "University Hospital North Durham Hospital",
```
"Durham" and "University Hospital North Durham Hospital" as the first line of the address. If there are multiple locations in Durham we could set the title to "Durham (University Hospital)".

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "ee399dec-f9c2-4201-b27f-e3da114d3e17"
+  },
+  {
+    "title": "Stockport Library",
```
``` json
"title": "Stockport",
"address": "Stockport Library\nWellington Rd South\nStockport\nSK1 3RS"
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "38fe9633-d556-4e96-ab99-518eeaf7d2a7"
+  },
+  {
+    "title": "Blackburn Library",
```
``` json
"title": "Blackburn",
"address": "Blackburn Library\nTown Hall Street\nBlackburn\nLancashire\nBB2 1AG"
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "c554946e-7b79-4446-b2cd-d930f668e54b"
+  },
+  {
+    "title": "Preston County Hall",
```
``` json
"title": "Preston",
"address": "Preston County Hall\nFishergate\nPreston\nLancashire\nPR1 8XJ"
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "c554946e-7b79-4446-b2cd-d930f668e54b"
+  },
+  {
+    "title": "Rothwell One Stop Centre",
```
``` json
"title": "Rothwell",
"address": "Rothwell One Stop Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "91b67ee6-f131-44d1-a05e-aac1306adba4"
+  },
+  {
+    "title": "Dewsbury Road One Stop Centre",
```
``` json
"title": "Leeds",
"address": "Dewsbury Road One Stop Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "91b67ee6-f131-44d1-a05e-aac1306adba4"
+  },
+  {
+    "title": "Cardiff Central Hub",
```
``` json
"title": "Cardiff Central",
"address": "Cardiff Central Hub\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "525da418-ff2c-4522-90a9-bc70ba4ca78b"
+  },
+  {
+    "title": "Llandaff North Library",
```
``` json
"title": "Llandaff North",
"address": "Llandaff North Library\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "525da418-ff2c-4522-90a9-bc70ba4ca78b"
+  },
+  {
+    "title": "Ammanford Library",
```
``` json
"title": "Ammanford",
"address": "Ammanford Library\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "b63e0799-2161-482f-bdfe-5188f904da32"
+  },
+  {
+    "title": "Chesterfield Library",
```
``` json
"title": "Chesterfield",
"address": "Chesterfield Library\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "fdcde285-37fd-4858-b00a-4ee31b4488b1"
+  },
+  {
+    "title": "Wychwood Surgery",
```
``` json
"title": "Chipping Norton",
"address": "Wychwood Surgery\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "2922aaf2-a3c9-4c89-8fe5-014c61199569"
+  },
+  {
+    "title": "Crowborough Community Centre",
```
``` json
"title": "Crowborough",
"address": "Crowborough Community Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "8e0e385f-2ec9-4e4f-b38e-63ca22d2ac1a"
+  },
+  {
+    "title": "Forest Row Community Centre",
```
``` json
"title": "Forest Row",
"address": "Forest Row Community Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "8e0e385f-2ec9-4e4f-b38e-63ca22d2ac1a"
+  },
+  {
+    "title": "Robertsbridge Hall",
```
``` json
"title": "Robertsbridge",
"address": "Robertsbridge Hall\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "8e0e385f-2ec9-4e4f-b38e-63ca22d2ac1a"
+  },
+  {
+    "title": "The Bloxham Centre",
```
``` json
"title": "Rugby",
"address": "The Bloxham Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "26f675df-fbc6-41a2-9594-485c2052ec2e"
+  },
+  {
+    "title": "Hatters Space Community Centre",
```
``` json
"title": "Nuneaton",
"address": "Hatters Space Community Centre\n ....."
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "26f675df-fbc6-41a2-9594-485c2052ec2e"
+  },
+  {
+    "title": "Citizens Advice Waverley (Godalming)",
```
``` json
"title": "Godalming",
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "df6fdf18-e089-49a1-9453-1400d55b96e9"
+  },
+  {
+    "title": "Age UK - Horsham",
```
``` json
"title": "Horsham"
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "4c1c3978-ffad-4a47-97ba-210e6eb8857f"
+  },
+  {
+    "title": "Gateshead Citizens Advice",
```
``` json
"title": "Gateshead",
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "bcfaa413-e6d3-4927-9f7c-47e97d00732c"
+  },
+  {
+    "title": "Rotterdam House",
```
``` json
"title": "Newcastle upon Tyne",
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "bcfaa413-e6d3-4927-9f7c-47e97d00732c"
+  },
+  {
+    "title": "Brigg Local Links",
```
``` json
"title": "Brigg",
```

andrewgarner commented on public/js/cita.json:
```diff
@@ -5008,5 +4868,305 @@
+    "booking_location_id": "bc588eed-fc08-4448-b793-a287e417c2be"
+  },
+  {
+    "title": "Pendleton Gateway",
```
``` json
"title": "Pendleton",
```

Commit:
Add missing newline before postcode

Commit:
Add missing coordinates for Stublee Hall in Lancashire

[Close to the edge..](https://www.google.co.uk/maps/place/Close+To+The+Edge/@53.6927199,-2.2040092,266m/data=!3m1!1e3!4m2!3m1!1s0x0000000000000000:0xf00c8a3791f43fb8)

Commit:
Reworded titles so that they just contain the location

Commit:
Fix Kirkby-in-Ashfield typo

Commit:
FixWare :hammer:

Commit:
Fix Poyton title

jiannis commented:
Think this is good enough. Shall we :rocket: it? 

[Close to the edge..](https://youtu.be/VjLHrhLVwzs?t=40s)

stevenwilkin commented:
![grandmaster flash](http://hiphopgoldenage.com/wp-content/uploads/2015/07/gal-gmf-8-jpg.jpg)

andrewgarner commented:
We've got some duplicate titles that we'll need to look at:
- [x] Stockport
- [x] Peterlee
- [x] Pendleton
- [x] Durham
- [x] Chipping Norton
- [x] Chesterfield
- [x] Bishop Auckland

jiannis commented:
Don't think duplicate titles is a problem. As a user I can distinguish between two different locations in the same area by looking at their address and then choosing the most convenient one.

The problem is why we have for example [two locations within a 6 min walk of each other](https://goo.gl/maps/iqVfs3ZzS9M2) in a town with a target audience of ~50,000 according to the [2011 census](http://www.stockport.gov.uk/2013/2978/stockporteconomicoverviewjune2013).

cc: @mcornford

Commit:
Remove duplicate location

Commit:
Remove duplicate location

jiannis commented:
Durham – a town with a target audience of ~10,000 according to the [2011 census](http://durobs.durham.gov.uk/ia/ia1008/) – also has [two locations within a 24 min walk of each other](https://goo.gl/maps/wsG2eGEpAWU2). Have renamed one to _Durham (University Hospital)_.

Commit:
Rename location to avoid duplicate title

Commit:
Let’s eat grandpa.

Commit:
Rename location

Commit:
Rename location to avoid duplicate title

Commit:
Rename location to avoid duplicate title

Commit:
Rename location to avoid duplicate title

jiannis commented:
@stevenwilkin :kiss:

stevenwilkin commented:
Me love you long time!
